specs/gloas/p2p-interface.md

```diff
@@ -318,7 +318,8 @@ The following validations MUST pass before forwarding the
 **Added in Gloas:**
 
 - _[IGNORE]_ The sidecar's `beacon_block_root` has been seen via a valid signed
-  execution payload header (builder's bid).
+  execution payload header (builder's bid). A client MAY queue the sidecar for
```

Comment:
Any specific reason to use the word "queue" then just cache?

Reply:
Same language as in previous forks